neurodamus/node.py

```diff
@@ -456,6 +456,16 @@ def compute_load_balance(self):
             if file_exists:
                 alloc = self._dry_run_stats.import_allocation_stats(filename, self._cycle_i)
             else:
+                if not (
+                    Path(DryRunStats._MEMORY_USAGE_FILENAME).exists()
+                    and Path(DryRunStats._MEMORY_USAGE_PER_METYPE_FILENAME).exists()
+                ):
+                    raise RuntimeError(
+                        f"No files {DryRunStats._MEMORY_USAGE_FILENAME} or "
```

Comment:
Nitpick: 
```suggestion
                        f"Missing {DryRunStats._MEMORY_USAGE_FILENAME} or "
```

could also make the change Weina suggested about the using `--dry-run` in the message

Reply:
done